tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/CookieTokenTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2016-2020 Contributors to the Eclipse Foundation
+ *
+ *  See the NOTICE file(s) distributed with this work for additional
+ *  information regarding copyright ownership.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.jwt.tck.container.jaxrs;
+
+import org.eclipse.microprofile.jwt.tck.util.MpJwtTestVersion;
+import org.eclipse.microprofile.jwt.tck.util.TokenUtils;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashSet;
+
+import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
+import static org.eclipse.microprofile.jwt.tck.TCKConstants.TEST_GROUP_JAXRS;
+
+public class CookieTokenTest extends Arquillian {
+    @ArquillianResource
+    private URL baseURL;
+
+    @Deployment
+    public static WebArchive createDeployment() throws IOException {
+        URL config = InvalidTokenTest.class.getResource("/META-INF/microprofile-config-cookie.properties");
+        URL publicKey = InvalidTokenTest.class.getResource("/publicKey.pem");
+        WebArchive webArchive = ShrinkWrap
+            .create(WebArchive.class, "CookieTokenTest.war")
+            .addAsManifestResource(new StringAsset(MpJwtTestVersion.MPJWT_V_1_2.name()), MpJwtTestVersion.MANIFEST_NAME)
+            .addAsResource(publicKey, "/publicKey.pem")
+            .addClass(TCKApplication.class)
+            .addClass(RolesEndpoint.class)
+            .addAsWebInfResource("beans.xml", "beans.xml")
+            .addAsManifestResource(config, "microprofile-config.properties");
+        System.out.printf("WebArchive: %s\n", webArchive.toString(true));
```

Comment:
I'm removing println in the new tests, I'll deal with the cleanup in the other tests later

Reply:
Yeah, I don't like them either, but I've added them to keep consistency. If the idea is to remove them in the other tests, I can remove these right away :)